source/MaterialXGenShader/ColorManagementSystem.h

```diff
@@ -25,6 +25,51 @@ class ShaderGenerator;
 /// A shared pointer to a ColorManagementSystem
 using ColorManagementSystemPtr = shared_ptr<class ColorManagementSystem>;
 
+/// Set of mappings between a uniform name and it's associated value
+using ColorSpaceUniformPtr = shared_ptr<class ColorSpaceUniform>;
```

Comment:
Uniforms for reflection and binding.
As discussed, will not try and move the Image class here, nor make it a new Value type.
Thus we have a different structure for constants vs textures.